### PR TITLE
[Python] Add django-ninja

### PR DESCRIPTION
**Pull request by vitalik**

### CODE REVIEW

File: python/django-ninja/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+django>=3.1.0,<3.2.0
+django-ninja==0.3.1
```
Comment (waghanza) [inline python/django-ninja/requirements.txt:2]: could you use a more relax constraint ? e.g ->  `django-ninja>=0.3.1,<0.4.0`

File: python/django-ninja/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+django>=3.1.0,<3.2.0
```
Comment (waghanza) [inline python/django-ninja/requirements.txt:1]: pining `django` is necessary here ?

File: python/django-ninja/config.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+framework:
+  website: http://django-ninja.rest-framework.com/
+  version: 0.3.1
```
Comment (waghanza) [inline python/django-ninja/config.yaml:3]: could you use `X.Y` format ? e.g : `0.3`

File: python/django-ninja/config.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+framework:
+  website: http://django-ninja.rest-framework.com/
```
Comment (waghanza) [inline python/django-ninja/config.yaml:2]: could you remote `http` -> will be added automatically

File: python/django-ninja/requirements.txt (hunk 2 above)
Comment (vitalik) [reply python/django-ninja/requirements.txt:1]: yes, 3.1+ comes with async views that are required

File: python/django-ninja/config.yaml (hunk 3 above)
Comment (vitalik) [reply python/django-ninja/config.yaml]: done

File: python/django-ninja/config.yaml (hunk 4 above)
Comment (vitalik) [reply python/django-ninja/config.yaml]: done

File: python/django-ninja/requirements.txt (hunk 1 above)
Comment (vitalik) [reply python/django-ninja/requirements.txt]: done

File: python/django-ninja/requirements.txt (hunk 2 above)
Comment (waghanza) [reply python/django-ninja/requirements.txt:1]: Is the 3.2 released ? 
I mean :
+ without any version pip will take the last version
+ this should not be done by `django-ninja` itself ?

Comment (vitalik) [reply python/django-ninja/requirements.txt]: @waghanza , indeed - updated requirements

File: python/django-ninja/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+django-ninja==0.6.0
```
Comment (waghanza) [inline python/django-ninja/requirements.txt:1]: better to use a range, no ?
`django-ninja>=0.6.0,<0.7.0`

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by vitalik** [comment]
@waghanza  done (well I set range till 1.0 as I expect only minor changes and bug fixes before 1.0)

File: python/django-ninja/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
+django-ninja>=0.6.0,<1.0.0
```
Comment (waghanza) [inline python/django-ninja/requirements.txt:1]: could you update to `>=0.8.0,<0.9.0` ?

File: python/django-ninja/config.yaml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+framework:
+  website: django-ninja.rest-framework.com
+  version: 0.3
```
Comment (waghanza) [inline python/django-ninja/config.yaml:3]: could you update to `0.8` ?

**Review by waghanza — CHANGES_REQUESTED** [review]
could you also rebase (I've changed CI) @vitalik ?

**Comment by vitalik** [comment]
done

File: python/django-ninja/config.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+framework:
+  website: django-ninja.rest-framework.com
+  version: 0.8.2
```
Comment (waghanza) [inline python/django-ninja/config.yaml:3]: could you use 0.8 instead of 0.8.2 ?

File: python/django-ninja/config.yaml (hunk 7 above)
Comment (vitalik) [reply python/django-ninja/config.yaml]: sure, done

**Review by waghanza — APPROVED** [review]
I don't know why CI is stuck, but this is OK imho

Thanks @vitalik.

May I add you to `CODEOWNER` in order to ping you on each `django-ninja` update `here` ?

**Pull request merged**